### PR TITLE
(docs) update README.md

### PR DESCRIPTION
**Pull request by vladdoster**
- fix grammar
- fix spelling
- fix punctuation
- fix split infinitives

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,24 +62,23 @@ Issues labeled [help wanted](https://github.com/photoprism/photoprism/labels/hel
...
-especially if you have feature requests, or need help with using our software.
+especially if you have feature requests or need help with using our software.
 In addition, you can find us on [Patreon](https://www.patreon.com/photoprism)
 and [PayPal](https://www.paypal.me/photoprism). 
 
 Your continuous support helps...
 
-* to pay for operating expenses and external services like satellite maps
-* developing new features, and keeping them free for everyone 🌈
+* To pay for operating expenses and external services like satellite maps
```
Comment (lastzero) [inline README.md:80]: Why uppercase?

Comment (vladdoster) [reply README.md:80]: To match the rest of the bullet points in the document.

Comment (vladdoster) [reply README.md:80]: Huh, let me revert this. Not sure how I missed the next bullet isn't either.

File: README.md [hunk 2]
```diff
@@ -32,22 +32,22 @@ To get a first impression, you're welcome to play with our public demo at [demo.
...
 
 Step-by-step installation instructions for our self-hosted [community edition](https://photoprism.app/get) can be found 
 on [docs.photoprism.org](https://docs.photoprism.org/getting-started/) -
-all you need is a Web browser and Docker to run the server. It is available for Mac, Linux and Windows.
+all you need is a Web browser and Docker to run the server. It is available for Mac, Linux, and Windows.
 
 We recommend hosting PhotoPrism on a server with **at least 2 cores** and **4 GB of memory**.
 Beyond these minimum requirements, the amount of RAM should match the number of cores.
-Indexing large photo and video collections significantly benefits from fast, local SSD storage.
+Indexing large photo and video collections significantly benefit from fast, local SSD storage.
```
Comment (lastzero) [inline README.md:46]: Indexing benefits... why benefit?

File: README.md [hunk 3]
```diff
@@ -91,12 +90,12 @@ Let us know if we mistakenly label an idea as [unfunded](https://github.com/phot
...
 *Note that we've recently moved GitHub Sponsors from [sponsors/lastzero](https://github.com/sponsors/lastzero) to an 
 [organization account](https://github.com/sponsors/photoprism) so that donations can be shared more easily.
-Tiers are more affordable now while benefits stay mostly the same.*
+Tiers are more affordable now, while benefits stay mostly the same.*
 
 ## Trademarks ##
 
 PhotoPrism® is a registered trademark of Michael Mayer. You may use it as required to describe 
-our software, run your own server, for educational purposes, but not for offering commercial 
+our software, run your server for educational purposes, but not for offering commercial
```
Comment (lastzero) [inline README.md:98]: These are two items: 1) run your server, 2) for educational purposes

Comma seems right?

**Comment by lastzero** [comment]
Thank you very much for improving our readme... my English is good, but not native :)

File: README.md [hunk 4]
```diff
@@ -62,24 +62,23 @@ Issues labeled [help wanted](https://github.com/photoprism/photoprism/labels/hel
...
+especially if you have feature requests or need help with using our software.
 In addition, you can find us on [Patreon](https://www.patreon.com/photoprism)
 and [PayPal](https://www.paypal.me/photoprism). 
 
 Your continuous support helps...
 
-* to pay for operating expenses and external services like satellite maps
-* developing new features, and keeping them free for everyone 🌈
+* pay for operating expenses and external services like satellite maps
+* developing new features and keeping them free for everyone 🌈
```
Comment (benmccann) [inline README.md:81]:
```suggestion
* develop new features and keep them free for everyone 🌈
```

File: README.md (hunk 2 above)
Comment (benmccann) [reply README.md:46]: "benefits" is correct

File: README.md [hunk 5]
```diff
@@ -19,7 +19,7 @@ To get a first impression, you're welcome to play with our public demo at [demo.
 
 ## Key Features ##
 
-* Our clearly structured [user interface](https://demo.photoprism.org/) makes browsing and organizing your personal photo collection as easy as 
+* Our structured [user interface](https://demo.photoprism.org/) makes browsing and organizing your photo collection as easy as
```
Comment (benmccann) [inline README.md:22]: this sounds strange to me with "clearly" removed

File: README.md [hunk 6]
```diff
@@ -62,24 +62,23 @@ Issues labeled [help wanted](https://github.com/photoprism/photoprism/labels/hel
...
 
-Follow us on [Twitter][twitter] to receive regular project updates and discuss development 
-related topics. Don't be afraid to ask stupid questions.
+Follow us on [Twitter][twitter] to receive regular project updates and discuss development related topics. Don't be afraid to ask stupid questions.
 
 ## Funding ##
 
 This software is about freedom and privacy but not necessarily about free beer. We feel like it
-would be a mistake to state there will be no costs, because clearly we have huge expenses, your server hardware
+would be a mistake to state there will be no costs because clearly, we have huge expenses, your server hardware
```
Comment (benmccann) [inline README.md:70]: I think it makes sense the comma that was there was removed, but I'm not sure why a new one was added

File: README.md [hunk 7]
```diff
@@ -91,12 +90,12 @@ Let us know if we mistakenly label an idea as [unfunded](https://github.com/phot
 
 *Note that we've recently moved GitHub Sponsors from [sponsors/lastzero](https://github.com/sponsors/lastzero) to an 
 [organization account](https://github.com/sponsors/photoprism) so that donations can be shared more easily.
-Tiers are more affordable now while benefits stay mostly the same.*
+Tiers are more affordable now, while benefits stay mostly the same.*
```
Comment (benmccann) [inline README.md:93]: I'm not sure why this comma was added either

File: README.md (hunk 4 above)
Comment (lastzero) [reply README.md:81]: Isn't developing a continuous process that requires the present participle?

Comment (benmccann) [reply README.md:81]: You would read this combined with the part before the list "Your continuous support helps develop new features and keep them free for everyone" vs "Your continuous support helps developing new features and keeping them free for everyone"

Comment (lastzero) [reply README.md:81]: We only pay our bills once a month, so not the same :wink:

**Comment by lastzero** [comment]
I'll merge this and fix remaining issues in develop - thanks :+1:

**Pull request merged**